components/config/src/lib.rs

```diff
@@ -50,6 +50,9 @@ pub struct Config {
     /// Whether to compile the `sass` directory and output the css files into the static folder
     pub compile_sass: Option<bool>,
 
+    /// Languages list and translated strings
+    pub languages: Option<HashMap<String, Toml>>,
```

Comment:
bikeshedding but what do you think about `translations` rather than `languages`? it would express that it's translation better I think but not 100% sure so only change if you think the same

Reply:
👍 I don’t mind bikeshedding when we are on the same page.
I’ll also change i18n() to trans() as discussed on rfc. Closer to Django and easier to type.